src/Lecture2.hs

```diff
@@ -62,7 +65,9 @@ lazyProduct = error "TODO"
 "ccaabb"
 -}
 duplicate :: [a] -> [a]
-duplicate = error "TODO"
+duplicate list = case list of
+     [var] -> [var, var]
+     _ -> head list : head list : duplicate (tail list)
```

Comment:
Try to use `[]` and `(x : xs)` patterns on `list` instead of `[var]` and the `_`

Reply:
Okay, I'll redo this for the lecture2 exercises PR. :)